pygpu/tests/test_elemwise.py

```diff
@@ -101,87 +100,6 @@ def ielemwise2_ops_array(op, dtype1, dtype2, shape):
     assert numpy.allclose(out_c, numpy.asarray(out_g), atol=1e-6)
 
 
-def test_elemwise_layouts():
-    for shape in [(), (20, 30), (50, 8, 9)]:
-        for offseted_outer in [True, False]:
-            for offseted_inner in [True, False]:
-                for sliced in [1, 2]:
-                    for order in ['c', 'f']:
-                        yield elemwise_layouts, shape, offseted_outer, \
-                            offseted_inner, sliced, order
-                        yield elemwise_layouts_mixed, shape, offseted_outer, \
-                            offseted_inner, sliced, order
-
-
-def test_elemwise_0():
-    elemwise_layouts((0,), False, False, 1, 'c')
```

Comment:
If we don't support it there, then we will need to support it at every places that call elemwise. I prefer to support it at only 1 place...